src/Logging/Logging.EventSource/test/EventSourceLoggerTest.cs

```diff
@@ -690,6 +690,18 @@ private static class EventTypes
             { "E5FM", (e) => VerifySingleEvent(e, "Logger2", EventTypes.FormattedMessage, 5, null, LogLevel.Critical,
                 @"""FormattedMessage"":""Logger2 Event5 Critical bar 23 45") },
 
+// Starting in netcoreapp3.0 Exception.ToString() puts a newline before inner exceptions
```

Comment:
@anurse Take a quick look at the change before the bot auto-merges :)

Reply:
Is there a way to use a single value and have some kind of "transform" applied in either `NETCOREAPP` or `!NETCOREAPP` instead of using separate values entirely?

Reply:
Yeah:
```csharp
$"some json string stuff, exception{GetNewLineOnNetCoreApp()} ---> inner exception"
```

```csharp
static string GetNewLineOnNetCoreApp()
{
#if NETCOREAPP
    return Environment.NewLine.Replace(...);
#else
    return "";
#endif
}
```